cmd/bridge/main.go

```diff
@@ -610,17 +604,25 @@ func main() {
 				http.Redirect(res, req, redirectURL.String(), http.StatusMovedPermanently)
 			})
 			redirectPort := fmt.Sprintf(":%d", *fRedirectPort)
-			log.Infof("Listening on %q for custom hostname redirect...", redirectPort)
-			log.Fatal(http.ListenAndServe(redirectPort, redirectServer))
+			klog.V(2).Infof("Listening on %q for custom hostname redirect...", redirectPort)
+			klog.Fatal(http.ListenAndServe(redirectPort, redirectServer))
 		}()
 	}
 
-	log.Infof("Binding to %s...", httpsrv.Addr)
+	klog.V(2).Infof("Binding to %s...", httpsrv.Addr)
 	if listenURL.Scheme == "https" {
-		log.Info("using TLS")
-		log.Fatal(httpsrv.ListenAndServeTLS(*fTlSCertFile, *fTlSKeyFile))
+		klog.V(2).Info("using TLS")
```

Comment:
Lets display this by default, as we are doing now
```suggestion
		klog.Info("using TLS")
```